pkg/rendering/helm.go

```diff
@@ -0,0 +1,61 @@
+package rendering
```

Comment:
I'd make this package exlicitly about helm rendering, instead of building one package for all future renderers.